quam_builder/builder/superconducting/build_quam.py

```diff
@@ -161,7 +161,8 @@ def add_octaves(machine: AnyQuam, calibration_db_path: Optional[Union[Path, str]
     AnyQuam: The QuAM with the added octaves.
     """
     if calibration_db_path is None:
-        calibration_db_path = machine.get_quam_state_path().parent
+        serialiser = machine.get_serialiser()
+        calibration_db_path = serialiser._get_state_path().parent
```

Comment:
What does this do exactly?
Where is supposed to be taken the calibration_db_path if not provided?

Reply:
Should be the parent class of quam_state, so `quam_config`